### ISSUE TITLE
chat bot is not working correctly:- The attention mask is not set and cannot be inferred from input because pad token is same as eos token.

### ISSUE THREAD

**Issue by Noor-sayyed** [Sushobhan55/DialoGPT-chatbot#1]
Friend: Hey!
>> You: heyyyy
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Friend: I love my wife a lot, she is the best.
>> You: hi
Friend: I'm sorry to hear that. I hope she feels better.
